server/webapp/WEB-INF/rails/webpack/views/components/forms/input_fields.tsx

```diff
@@ -291,3 +292,56 @@ export class SelectFieldOptions extends MithrilViewComponent<SelectFieldAttrs> {
     });
   }
 }
+
+export interface RadioButtonAttrs<T> {
+  label: string;
+  errorText?: string;
+  disabled?: boolean;
+  required?: boolean;
+  property: (newValue?: T) => T;
+  possibleValues: Map<string, T>;
+}
+
+export class RadioField<T> extends MithrilViewComponent<RadioButtonAttrs<T>> {
```

Comment:
You can do this using `FormField` see [Switch Component](https://github.com/gocd/gocd/blob/94e56f010804b99215bd78248848fe16eac4596c/server/webapp/WEB-INF/rails/webpack/views/components/forms/input_fields.tsx#L234) for the reference.